packages/shorebird_cli/lib/src/shorebird_process.dart

```diff
@@ -50,7 +50,8 @@ abstract class ShorebirdProcess {
 
   static String _resolveExecutable(String executable) {
     if (executable == 'flutter') {
-      return ShorebirdPaths.flutterBinaryFile.path;
+      return 'FLUTTER_STORAGE_BASE_URL="https://download.shorebird.dev/" '
```

Comment:
You don't want to do this this way, because this depends on runInShell=true, which you may not want in all environments.

Instead, you will want to modify the env passed to the call.

Reply:
https://github.com/shorebirdtech/shorebird/blob/main/packages/shorebird_cli/lib/src/shorebird_process.dart#L25

https://api.dart.dev/stable/2.10.5/dart-io/Process/run.html
says you can just set the one env and it will inherit by default.